src/commands/flow/internal/spec/checkPredecessorDeadlock.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { checkPredecessorDeadlock } from '..'
+
+describe('checkPredecessorDeadlock', () => {
+  it('should return true, if predecessorDeadlock is present and itself', () => {
```

Comment:
`should return true, if predecessorDeadlock is present and itself` -> `should return true, if predecessorDeadlock is present and it is pointing to itself`

Reply:
fixed